tool/src/main.rs

```diff
@@ -5,13 +5,16 @@ extern crate serde;
 extern crate byteorder;
 #[cfg(windows)] extern crate winapi;
 #[cfg(windows)] extern crate kernel32;
+extern crate nfd;
 
 #[macro_use] mod error;
 mod tas;
 mod config;
 #[cfg(windows)] mod inject;
 
+use nfd::Response;
```

Comment:
I usually organize my imports in this order: std, crates, modules, each separated with a newline.